src/field_effect_helpers.c

```diff
@@ -344,7 +346,8 @@ u32 FldEff_Shadow(void)
     graphicsInfo = GetObjectEventGraphicsInfo(gObjectEvents[objectEventId].graphicsId);
     if (graphicsInfo->shadowSize == SHADOW_SIZE_NONE) // don't create a shadow at all
         return 0;
-    spriteId = CreateSpriteAtEnd(gFieldEffectObjectTemplatePointers[sShadowEffectTemplateIds[graphicsInfo->shadowSize]], 0, 0, 0x94);
+    LoadSpriteSheetByTemplate(gFieldEffectObjectTemplatePointers[sShadowEffectTemplateIds[graphicsInfo->shadowSize]], 0);
+    spriteId = CreateSpriteAtEnd(gFieldEffectObjectTemplatePointers[sShadowEffectTemplateIds[graphicsInfo->shadowSize]], 0, 0, 0x94 + 1); // higher = farther back; shadows should be behind object events
```

Comment:
I'm not a fan of the addition of hexadecimal and decimal

Reply:
Hm. Below I thought `0x94` was the subpriority of shadows, but based on this comment it's the subpriority of object events? `OW_OBJECT_SUBPRIORITY + 1` would communicate that better.

I'm also a bit concerned that [`SetObjectSubpriorityByElevation`](<https://github.com/rh-hideout/pokeemerald-expansion/blob/master/src/event_object_movement.c#L9308-L9316>) exists and doesn't seem to set the subpriority to a flat `0x94` unless ` tmp3 + sElevationToSubpriority[elevation] + subpriority` somehow always works out as that. My guess is that `0x94` is the maximum subpriority that this expression can compute, and so `0x94 + 1` means the sprite will always be below all objects; whereas the current `0x94` could sometimes put the shadow above the object based on their Y coordinates?

It's also worth noting that subpriorities are already written in decimal in [`sElevationToSubpriority`](<https://github.com/rh-hideout/pokeemerald-expansion/blob/master/src/event_object_movement.c#L9243-L9245>) so it should probably be `148` not `0x94` anyway.